.pre-commit-config.yaml

```diff
@@ -1,13 +1,17 @@
 repos:
 -   repo: https://github.com/ambv/black
-    rev: 18.6b4
+    rev: 18.9b0
```

Comment:
Can you also update the version from `tox.ini`?

Reply:
Sure.